packages/react-core/src/next/components/Modal/examples/Modal.md

```diff
@@ -0,0 +1,157 @@
+---
+id: Modal
+section: components
+cssPrefix: pf-v5-c-modal-box
+propComponents: ['Modal', 'ModalBoxBody', 'ModalBoxHeader', 'ModalBoxFooter']
+ouia: true
+beta: true
+---
+
+import WarningTriangleIcon from '@patternfly/react-icons/dist/esm/icons/warning-triangle-icon';
+import CaretDownIcon from '@patternfly/react-icons/dist/esm/icons/caret-down-icon';
+import BullhornIcon from '@patternfly/react-icons/dist/esm/icons/bullhorn-icon';
+import HelpIcon from '@patternfly/react-icons/dist/esm/icons/help-icon';
+import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
+import formStyles from '@patternfly/react-styles/css/components/Form/form';
+
+## Examples
+
+### Basic modals
+
+Basic modals give users the option to either confirm or cancel an action. To flag an open modal, use the `isOpen` property. To execute a callback when a modal is closed, use the `onClose` property.
+
+```ts file="./ModalBasic.tsx"
+
+```
+
+### Scrollable modals
+
+To enable keyboard-accessible scrolling of a modal’s content, pass `tabIndex={0}` to the `<ModalBoxBody>`.
+
+```ts file="ModalWithOverflowingContent.tsx"
+
+```
+
+### With a static description
+
+To provide additional information about a modal, use the `description` property. Descriptions are static and do not scroll with other modal content.
+
+```ts file="./ModalWithDescription.tsx"
+
+```
+
+### Top aligned
+
+To override a modal's default center alignment, use the `position` property. In this example, `position` is set to "top", which moves the modal to the top of the screen.
+
+```ts file="./ModalTopAligned.tsx"
+
+```
+
+### Small modal
+
+To adjust the size of a modal, use the `variant` property. Modal variants include "small", "medium", "large", and "default".
+
+The following example displays a "small" modal by passing in `variant={ModalVariant.small}`.
+
+```ts file="./ModalSmall.tsx"
+
+```
+
+### Medium modal
+
+The following example displays a "medium" modal by passing in `variant={ModalVariant.medium}`.
+
+```ts file="./ModalMedium.tsx"
+
+```
+
+### Large modal
+
+The following example displays a "large" modal by passing in `variant={ModalVariant.large}`.
+
+```ts file="./ModalLarge.tsx"
+
+```
+
+### Custom width
+
+To choose a specific width for a modal, use the `width` property. The following example has a `width` of "50%".
+
+```ts file="./ModalCustomWidth.tsx"
+
+```
+
+### Custom header and footer
+
+To add a custom header and footer to a modal, set the `header` and `footer` properties to a custom implementation. The following example passes title components into both the header and the footer and also passes an icon to the footer.
+
+```ts file="./ModalCustomHeaderFooter.tsx"
+
+```
+
+### No header or footer
+
+To exclusively present information in a modal, remove the `header` and/or `footer`.
+
+```ts file="./ModalNoHeaderFooter.tsx"
+
+```
```

Comment:
I left both examples.  I Changed `Custom header and footer` to `Custom header` since there is nothing to pass to footer as a prop even when not custom.

I can remove the "No header or footer example if we think it adds no value.  Core however still has that exmple.

Reply:
I think mentioning that ModalBoxHeader and ModalBoxFooter in the basic example sounds useful regardless. I'll add a suggestion abt that

It seems fine to keep/modify this example since it's calling out a specific situation/prop (title) that doesn't seem to be mentioned anywhere else already. Plus it parallels to the previous non-Next example?

But FYI the button in the example still says "header/footer", so it would need to be updated to just "header"!